src/sentry/web/frontend/setup_wizard.py

```diff
@@ -143,6 +133,80 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
         context["organizations"] = list(org_mappings_map.values())
         return render_to_response("sentry/setup-wizard.html", context, request)
 
+    def post(self, request: HttpRequest, wizard_hash=None) -> HttpResponse:
+        """
+        This updates the cache content for a specific hash
+        """
+        organization_id = request.POST.get("organizationId", None)
+        project_id = request.POST.get("projectId", None)
+
+        if organization_id is None or project_id is None or wizard_hash is None:
+            return HttpResponseBadRequest()
```

Comment:
Not sure if I returned the correct response objects every time. 🤔 
Tried to copy what I saw in other endpoints.

Reply:
This is ok, there are multiple ways, but we do not have some standard way of doing this :)

You can either raise exception (e.g. `Http404`), which will then be caught by Django and transformed into a response, or just return response with status code you want.